app/services/auth_service.py

```
@@ -378,67 +380,117 @@ def verify_phone(
+            # make sure media/photos exists
+            os.makedirs(file_abs_path.parent, exist_ok=True)
+
+            # Open the destination file in binary write mode (async)
+            async with aiofiles.open(file_abs_path, "wb") as dest_file:
+                while chunk := await file.read(8192):
+                    await dest_file.write(chunk)
 
-            # Check if email is already in use
-            existing_email_user = (
-                self.db.query(User).filter(User.email == email, User.id != user.id).first()
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)